hack/e2e-debug.sh

```diff
@@ -35,4 +35,4 @@ wait_until_pods_running knative-eventing || fail_test "Pods in knative-eventing
 
 header "Running tests"
 
-go_test_e2e -timeout=30m -run="${test_name}" "${test_dir}" || fail_test "Test(s) failed"
+go test -tags=e2e -v -timeout=30m -run="${test_name}" "${test_dir}" || fail_test "Test(s) failed"
```

Comment:
Is there a reason for this change?

Reply:
`go_test_e2e` is not very human friendly as it's meant for prow/machines